src/page-multi-facility/ReferenceDataModal/SyncNoUserFacilities.tsx

```diff
@@ -0,0 +1,319 @@
+import dataSyncIcon from "../../design-system/icons/ic_data_sync.svg";
+import InputButton from "../../design-system/InputButton";
+import InputSelect from "../../design-system/InputSelect";
+import Modal from "../../design-system/Modal";
+import { useFacilities } from "../../facilities-context";
+import { LocaleData } from "../../locale-data-context";
+import { useLocaleDataState } from "../../locale-data-context";
+import useScenario from "../../scenario-context/useScenario";
+import SystemTypeSelection from "../SystemTypeSelection";
+import {
+  Facility,
+  FacilityReferenceMapping,
+  ReferenceFacility,
+} from "../types";
+import ReferenceFacilityRow from "./shared/ReferenceFacilityRow";
+
+const STATE_AND_SYSTEM_SELECTION_CARD = "STATE_AND_SYSTEM_SELECTION_CARD";
+const SYNC_REFERENCE_FACILITIES_CARD = "SYNC_REFERENCE_FACILITIES_CARD";
+
+const CardContent = styled.div`
+  color: ${Colors.green};
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: -0.02em;
+  line-height: 150%;
+`;
+
+const CardDescription = styled.div`
+  margin: 20px 0px;
+`;
+
+const CardNavigation = styled.div`
+  border-top: 0.5px solid ${Colors.darkGray};
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row-reverse;
+  padding-top: 20px;
+  width: 100%;
+`;
+
+const StateAndSystemForm = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+`;
+
+type StateNameProps = {
+  stateName?: string;
+  setStateName: (stateName?: string) => void;
+};
+
+type SystemTypeProps = {
+  systemType?: string;
+  setSystemType: (systemType?: string) => void;
+};
+
+type CardStateProps = {
+  setActiveStep: (activeStep: string) => void;
+  setModalTitle: (modalTitle: string) => void;
+};
+
+type StateNameSelectionProps = StateNameProps & {
+  stateNames: string[];
+};
+
+const StateNameSelection: React.FC<StateNameSelectionProps> = (props) => {
+  const { stateName, stateNames, setStateName } = props;
+
+  return (
+    <InputSelect
+      label="State"
+      value={stateName}
+      onChange={(event) => {
+        setStateName(event.target.value);
+      }}
+    >
+      <option value="" />
+      {stateNames.map((name) => (
+        <option key={name} value={name}>
+          {name}
+        </option>
+      ))}
+    </InputSelect>
+  );
+};
+
+type StateAndSystemProps = StateNameProps & SystemTypeProps & CardStateProps;
+
+const StateAndSystemSelectionCard: React.FC<StateAndSystemProps> = (props) => {
+  const {
+    stateName,
+    setStateName,
+    systemType,
+    setSystemType,
+    setActiveStep,
+    setModalTitle,
+  } = props;
+
+  // This card doesn't have a title so we just clear out any existing
+  // title that may exist if the user navigates back to this card.
+  setModalTitle("");
+
+  const { data: localeDataSource } = useLocaleDataState();
+
+  const stateNames = Array.from(localeDataSource.keys()).filter(
+    (name) => name !== "US Total",
```

Comment:
I'm currently still including "US Federal Prisons" in this list, but it's unclear if I should be